nf_core/modules/modules_command.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, dir, remote_url=None, branch=None, no_pull=False, base_path=N
         except LookupError as e:
             raise UserWarning(e)
 
-    def get_pipeline_modules(self):
+    def get_pipeline_modules(self, retrieve_local=False):
```

Comment:
Since we only use the modules from a single folder in the test command, it might make sense to implement a separate method for retrieving modules from single folder. It might make sense to have a separate method for local modules also, since they have such a different folder structure.

Reply:
I'm fine with writing a separate method for this, but in the case of running tests the folder structure won't change (see my comment above), might be useful in the future for other commands though 🤔 

I am really unsure about the best way to do that, local modules are such a special case! I also thought about retrieving both local and nf-core modules at the same time, but then the sanity check of folders must be different.